base_station/gui/src/conf.json

```diff
@@ -1,14 +1,14 @@
 {
 	"cameras":
 	{
-		"0": "Microscope",
+        "-1": "No Camera",
+		"0": "Camera 0",

```

Comment:
Can you fix the formatting and add a new line to this file?

Reply:
was this resolved?